notebooks/official/tabnet/tabnet_vertex_tutorial.ipynb

```diff
@@ -0,0 +1,972 @@
+{
```

Comment:
<p>Please try to programmatically extract the JOB_ID for the user. This approach will also fail when running our CI process.</p><p> </p><p>This <a href="https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/vertex_endpoints/tf_hub_obj_detection/deploy_tfhub_object_detection_on_vertex_endpoints.ipynb" target="_blank">notebook</a> provides a good example of extracting the <strong>MODEL_ID</strong> and <strong>ENDPOINT_ID</strong> using grep / sed, that you can apply here.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/375/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>Done</p>